tests/inc/options/test-class-wpseo-options.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+class WPSEO_Options_Test extends WPSEO_UnitTestCase {
+
+	public function setUp() {
+		parent::setUp();
+		WPSEO_Options::get_instance();
+	}
+
+	/**
+	 * Tests if the get_all function returns an array with at least the indexses associated with the options: wpseo, wpseo-rss and wpseo_internallinks.
+	 *
+	 * @covers WPSEO_Options::get_all
+	 */
+	public function test_get_all_HAS_VALID_KEYS() {
+		$result = WPSEO_Options::get_all();
```

Comment:
Change `WPSEO_Options::get_all()` into ``WPSEO_Options::get_options( self::get_option_names() );`` allows you to remove the method `get_all`.